### PR TITLE
Updated italian phrases for 1.14.6 release.

### PR DESCRIPTION
**Pull request by MishaBoar**
Minor update since last version to the new base language files for 1.14.6. Fixed a couple of phrases from the previous translation, too.

Windows and Linux build working fine.

Thank you!

### CODE REVIEW

File: src/qt/locale/bitcoin_it.ts [hunk 1]
```diff
@@ -4546,19 +4460,19 @@ p, li { white-space: pre-wrap; }
...
+        <translation>Esegui il comando quando viene rilevato un fork molto lungo (%s in cmd è sostituito dal messaggio)</translation>
     </message>
     <message>
         <source>Specify directory where to write backups and data dumps (default datadir/backups)</source>
-        <translation type="unfinished"></translation>
+        <translation>Specifica la cartella in cui salvare i backup e i dump di dati (default datadir/backups)</translation>
     </message>
 </context>
-</TS>
+</TS>
```
Comment (patricklodder) [inline src/qt/locale/bitcoin_it.ts:4478]: Could you please bring the newline back? Other than that, I'm ok with this.

**Comment by patricklodder** [comment]
ACK from me.

@MishaBoar could you please squash the 2 commits into 1? Thanks.

**Comment by MishaBoar** [comment]
Thanks Patrick - did I do this correctly? Thanks.

**Comment by patricklodder** [comment]
Yes, correct! Thanks!

**Review by patricklodder — APPROVED** [review]
ACK - technical review only. If no comments in the next 24h we can merge this, and any corrections coming in later can open a separate PR.

**Pull request merged**